### ISSUE TITLE
[1.0, meta] creditNotation、2選択肢なのでboolにしますか？

### ISSUE THREAD

**Issue by FMS-Cat** [vrm-c/vrm-specification#258]
https://github.com/vrm-c/vrm-specification/blob/b4bd1d5249072b2c4cfce8c273128fc89795f19d/specification/VRMC_vrm-1.0_draft/schema/VRMC_vrm.meta.schema.json#L87-L96

2選択肢なので、key名を `requireCreditNotation` にしてboolにしてしまっても良くないか……と思ったのですが、
他のboolの、trueが「許可する」を意味するのに対して、こちらはtrueが「要求する」を意味するものになるため、若干悩んでおります。

**Comment by FMS-Cat** [comment]
@ousttrue Metaの件で相談したいことがあります。よろしくおねがいします。

**Comment by ousttrue** [comment]
vrm-0 は、
bool はデフォルト値を false で統一した方が、プログラムは組みやすいです。
enum にする場合は、先頭がデフォルト値になるように列挙すると、プログラムは組みやすいです。
という決め方をしていた部分があるのですが、

vrm-1 は、`bool?` 型を使う方針なのでどっちでも OK です。falseの時に二重否定にならないような、シンプルな命名がいいと思います。

@FMS-Cat

**Comment by FMS-Cat** [comment]
そうですね。二重否定を避けると `isCreditNotationUnnecessary` とかですが、長いし分かりづらいですね……
